photonai/optimization/smac/smac.py

```diff
@@ -33,8 +33,7 @@ def __init__(self, facade=None, scenario_dict=None, intensifier_kwargs=None, rng
             msg = "Module smac not found or not installed as expected. " \
                   "Please install the smac_requirements.txt PHOTON provides."
             logger.error(msg)
-            raise ModuleNotFoundError("Module smac not found or not installed as expected. "
-                                      "Please install the smac_requirements.txt PHOTON provides.")
+            raise ModuleNotFoundError(msg)
```

Comment:
Log Output with PHOTON Logger so that error message is in the photon.log when console output is not available.